src/settings/90-docker.settings.inc

```diff
@@ -0,0 +1,40 @@
+<?php
+
+/**
+ * @file
+ * Drupal 8 settings for docker.
+ */
+
+if (SETTINGS_ENVIRONMENT == 'local' && SETTINGS_HOSTING == 'docker') {
+  $databases['default']['default'] = [
+    'database' => getenv('DB_NAME'),
+    'username' => getenv('DB_USER'),
+    'password' => getenv('DB_PASSWORD'),
+    'prefix' => '',
+    'host' => getenv('DB_HOST'),
+    'port' => '3306',
+    'driver' => getenv('DB_DRIVER'),
+  ];
+
+  $https = TRUE;
+
+  if (!isset($_SERVER['HTTPS']) || $_SERVER['HTTPS'] != 'on') {
```

Comment:
Not necessarily, as https detection may be environment specific. Pantheon uses $_SERVER['HTTP_X_SSL'] for example.